elements/pfe-clipboard/src/pfe-clipboard.scss

```diff
@@ -0,0 +1,129 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: clipboard;
+
+$pfe-clipboard--BackgroundColor--focus: rgba(40, 151, 240, 0.2);
```

Comment:
Since this only has 1 reference below, feel free to put it directly in the map instead of using a sass variable, just for simplicity.

Reply:
Removed, unused. 🪓